test/unit/Minter.js

```diff
@@ -281,27 +281,52 @@ contract("Minter", accounts => {
         })
     })
 
-    describe("depositETH", () => {
-        it("should fail if caller is not currently registered Minter or JobsManager", async () => {
-            await expectThrow(minter.depositETH({from: accounts[1], value: 100}))
+    describe("trustedDepositETH", () => {
+        it("should fail if caller is not currently registered Minter or TicketBroker", async () => {
+            await expectThrow(minter.trustedDepositETH({from: accounts[1], value: 100}))
         })
 
         it("should receive ETH from currently registered Minter", async () => {
             // Register mock Minter
             const mockMinter = await fixture.deployAndRegister(GenericMock, "Minter")
-            // Call depositETH on this Minter from currently registered Minter
-            await mockMinter.execute(minter.address, functionSig("depositETH()"), {from: accounts[1], value: 100})
+            // Call trustedDepositETH on this Minter from currently registered Minter
+            await mockMinter.execute(minter.address, functionSig("trustedDepositETH()"), {from: accounts[1], value: 100})
 
             assert.equal(await web3.eth.getBalance(minter.address), 100, "wrong minter balance")
         })
 
-        it("should receive ETH from JobsManager", async () => {
-            await fixture.jobsManager.execute(minter.address, functionSig("depositETH()"), {from: accounts[1], value: 100})
+        it("should receive ETH from TicketBroker", async () => {
+            await fixture.ticketBroker.execute(minter.address, functionSig("trustedDepositETH()"), {from: accounts[1], value: 100})
 
             assert.equal(await web3.eth.getBalance(minter.address), 100, "wrong minter balance")
         })
     })
 
+    describe("trustedBurnETH", () => {
```

Comment:
Perhaps we should add a test that fails when the system is paused, per the modifier on `trustedBurnETH` ?

Reply:
Added a bunch of tests for failure when the Controller is paused in `Minter.js`